packages/environment-check/package.json

```diff
@@ -0,0 +1,49 @@
+{
+    "name": "@sap-ux/environment-check",
+    "version": "0.0.1",
+    "description": "SAP Fiori environment check",
+    "license": "Apache-2.0",
+    "bin": {
+        "envcheck": "bin/envcheck"
+    },
+    "main": "dist/index.js",
+    "private": false,
+    "scripts": {
+        "envcheck": "./bin/envcheck",
```

Comment:
What is the reason for the this script? Is it to manually test that the exposed CLI works? If yes, could we rename it to sth like `test:bin`

Reply:
Updated